src/pages/packaging/index.tsx

```diff
@@ -243,11 +244,16 @@ const Page = () => {
   const packagingMaterials = useOptions("packaging_materials", lang);
   const packagingShapes = useOptions("packaging_shapes", lang);
   const packagingRecycling = useOptions("packaging_recycling", lang);
+  const [searchState] = useUrlParams({
+    country: "en:france",
```

Comment:
I would default this to world

Reply:
The reason is to do as many as possible products for the french agency ADEM and open it to the world after that

It's kind of related to the issue asking to add a field country in the settings such that